src/main/kotlin/nl/avisi/anowalke/toolWindow/FluffyListener.kt

```diff
@@ -0,0 +1,18 @@
+package nl.avisi.anowalke.toolWindow
+
+import com.intellij.openapi.fileChooser.FileChooser
+import com.intellij.openapi.fileChooser.FileChooserDescriptor
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.wm.ex.ToolWindowManagerListener
+import javax.swing.JLabel
+
+class FluffyListener(val project: Project): ToolWindowManagerListener {
+
+    override fun stateChanged() {
+    }
+
+    fun scanProject(currentClass: JLabel) {
+        val fileChooserDescriptor = FileChooserDescriptor(true, false, false,false, false, false)
```

Comment:
Het kan helpen om alle boolean-waarden in variabelen te zetten. Het is nu in eerste opzicht erg onduidelijk waar al die booleans voor staan.